packages/react-theme-provider/src/useThemeProviderClasses.tsx

```diff
@@ -1,9 +1,16 @@
+/* eslint-disable @typescript-eslint/naming-convention */
 import { makeClasses } from './makeClasses';
 import { tokensToStyleObject } from './tokensToStyleObject';
 
+const inheritFont = { fontFamily: 'inherit' };
+
 export const useThemeProviderClasses = makeClasses(theme => {
   const { tokens } = theme;
 
+  const bodyStyles = {
+    background: 'var(--body-background)',
```

Comment:
color should be applied?

Reply:
actually shouldn't the font etc all be defined here?

Reply:
Then:

```js
{
  _applyTo_element: bodyStyles,
  _applyTo_body: {
    'global(body)': bodyStyles
  }
}
```

Reply:
(and maybe rename it to `themeStyles` or something more general.

Reply:
oh i don't think that would work

Reply:
So I have styles which should be by default apply to `ThemeProvider` root defined in `root` slot. if `applyTo` is set to `element` or `body`, `bodyStyles` will be applied to `ThemeProvider`.

Applying styles to body is handled separately in `useApplyThemeToBody` hook, which handles applying the applicable classNames (`root-*` and `_applyTo_body-*`) to body element